packages/drupal/gutenberg_blocks/package.json

```diff
@@ -5,10 +5,14 @@
   "scripts": {
     "dev": "vite build --watch",
     "prep": "vite build && cp node_modules/@custom/ui/build/gutenberg.css css/ui.css",
-    "test:static": "tsc --noEmit && eslint \"**/*.{ts,tsx,js,jsx}\" --ignore-path=\"./.eslintignore\""
+    "test:static": "tsc --noEmit && eslint \"**/*.{ts,tsx,js,jsx}\" --ignore-path=\"./.eslintignore\"",
+    "generate-block": "node ./scripts/generate-gutenberg-block.js DemoBlock"
```

Comment:
Scripts can take arguments. That way it could actually be used.

Reply:
Thanks, updated this now.